docs/resources/resources.md

```diff
@@ -38,6 +39,7 @@ _These articles reflect the state of ZIO at the time of their publication. The c
 
 ## Sample Projects
 
+- [STM Partitioning - Code for the 'Exploring the STM functionality in ZIO' blog post)(https://github.com/freskog/stm-partitioning)
```

Comment:
@freskog I just noticed a typo here, there is `)` instead of `]`. Can you fix it in a new PR?